### PR TITLE
set up material-ui

### LINKED ISSUE #15: Material UI theme
# Project setup
- Configuration of Theme Provider for Material-UI:
  * Install @material-ui/core.
  * Using createMuiTheme to create custom theme variables (colors, breakpoints, typography ).
  * Wrapping ThemeProvider around app components.

### PR DESCRIPTION
**Pull request by AdhamHaisami**
Fixes #15
-  install `material-UI/core` package 
- customizing theme by adding theme.js file in `client/app/theme.js`
-  wrapping the app component with `themeProvider`

### CODE REVIEW

**Comment by AdhamHaisami** [comment]
Fix #50

File: client/src/App/theme.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const theme = createMuiTheme({
+  palette: {
+   type: 'dark',
```
Comment (imansedky5) [inline client/src/App/theme.js:5]: why did you added  ` type: 'dark'` when you added  `background-color: black;` in index.css

File: client/public/index.html [hunk 2]
```diff
@@ -2,16 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (mFlaifel) [inline client/public/index.html:5]: restore favicon
create new favicon with site logo

File: client/public/index.html [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="beauty salon application that helps customers reserve a date, and buy cosmetics."
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (mFlaifel) [inline client/public/index.html:12]: restore this file

File: client/src/App/theme.js (hunk 1 above)
Comment (AdhamHaisami) [reply client/src/App/theme.js:5]: type is only for material-UI components, Wile background-color in CSS file for the entire page

**Review by yasmeenattallah — APPROVED** [review]

**Review by mFlaifel — APPROVED** [review]
well done

**Review by khamis-hesham — APPROVED** [review]

**Review by yasmeenattallah — APPROVED** [review]

**Review by yasmeenattallah — APPROVED** [review]

**Review by imansedky5 — APPROVED** [review]

**Pull request merged**